python/triton/runtime/jit.py

```diff
@@ -77,6 +80,19 @@ def is_triton_builtin(func):
             key = func_cache_key + noinline
             self.hasher.update(key.encode("utf-8"))
 
+    def visit_FunctionDef(self, node):
+        # Save the local name which may hide the global name.
+        self.local_names = [arg.arg for arg in node.args.args]
+        self.generic_visit(node)
+
+    def visit_Assign(self, node):
+        _names = []
+        for target in node.targets:
+          _names += [target.id]
+        if len(_names) == 1:
+          self.local_names += _names
+        else:
+          raise TypeError(f"Simultaneous multiple assignment is not supported.")
```

Comment:
Thanks!

Just one minor comment.

A more fitting exception might be ValueError? Up to you.

Also I think f-string is not needed. Pleased drop and just use a string.